### ISSUE TITLE
[Bug] (SSH) Failed to validate host key. when using SSH Tunnel

### ISSUE THREAD

**Issue by dotob** [hankinsoft/SQLPro#151]
There is a problem with ssh host key validation. I have a Postgresql connection configured with a SSH Tunnel. The SSH Tunnel config uses a SSH key instead of password auth. 

The following is reproducable:
- no entry in .ssh/known_hosts
- SQLPro works 
- connect via commandline to the host, known_hosts has an entry for the host now
- SQLPro does not work anymore and gives the error '(SSH) Failed to validate host key.' in the query field or as a popup when connecting
- removing the host from known_hosts and SQLPro is working again

Log file says:
2017/10/12 08:39:05:792  Failed to connect to the SSH server because the key in the known_hosts file doesn't match.
2017/10/12 08:39:05:792  SSH tunnel failed with error: (SSH) Failed to validate host key.

**Comment by hankinsoft** [comment]
Hi,
Are you using SQLPro for Postgres or SQLPro Studio? I've just pushed a build for studio yesterday that fixes this (http://sqlprostudio.com/download.php). Please give that a shot and let me know if that helps out.

**Comment by dotob** [comment]
yes studio. i will check it and tell if it works. thanks

**Comment by pprazzi** [comment]
I have the same problem with mysql - exactly the same lines in my Log file

**Comment by Andware** [comment]
Yes the fix the problem with MySQL login and ssh key. The problem are, the SetApp version of SQLPro Studio is only 1.0.145. Can you update this to 1.0.151?

**Comment by dotob** [comment]
yes it is working here too. thanks for the great work
